wiggum_app/config.py

```diff
@@ -1,3 +1,4 @@
 DEBUG=True
 SECRET_KEY="4zNNYuWQjNCHoCFrhEKqUDLB1Z5xGl2W/Yliy63tD8po9qOQ0YjPum5OaSDnPvNvB1eLzany73EHJfFlYEov1g=="
 SERVER_NAME="127.0.0.1:5000"
+SAVE_FOLDER="data/"
```

Comment:
does this / resolve correctly on windows and mac?

Reply:
No, that is moving the hard code directory from controller.py to config.py.

Reply:
then #95 is not closed, becase  you are making the config as a place the user should do some setup, so:
1. make it a template and provide instructions for the user
2. make sure that where you use this you use something that safely puts the `/` or `\` as appropriate per os (eg `os.path.join`